tests/ci_tests/fixtures/prepare_environment_fixture.py

```diff
@@ -50,40 +55,154 @@ def _setup_database(db_conn):
+    role_name = _create_sagemaker_role(iam_client)
+    policy_arn = _create_sagemaker_policy(iam_client)
+    _attach_policy_to_role(iam_client,
+                           policy_arn=policy_arn,
+                           role_name=role_name)
+    _attach_policy_to_role(iam_client,
+                           policy_arn="arn:aws:iam::aws:policy/AmazonSageMakerFullAccess",
+                           role_name=role_name)
+    return role_name
+
+
+def _attach_policy_to_role(iam_client, policy_arn, role_name):
+    response = iam_client.attach_role_policy(
+        PolicyArn=policy_arn,
+        RoleName=role_name,
+    )
+
+
+def _create_sagemaker_role(iam_client):
+    role_name = "ci-exasol-sagemaker-extension-role"
+    try:
+        assume_policy_document = cleandoc("""
+        {
+            "Version": "2012-10-17",
+            "Statement": [
+                {
+                    "Effect": "Allow",
+                    "Principal": {
+                        "Service": "sagemaker.amazonaws.com"
+                    },
+                    "Action": "sts:AssumeRole"
+                }
+            ]
+        }
+        """)
+        response = iam_client.create_role(
+            RoleName=role_name,
+            AssumeRolePolicyDocument=assume_policy_document,
+            Description='This role is used for the CI Tests of the exasol.sagemaker-extension',
+        )
+    except iam_client.exceptions.EntityAlreadyExistsException as ex:
+        print("'EntityAlreadyExistsException' exception is handled")
+    return role_name
+
+
+def _create_sagemaker_policy(iam_client) -> str:
+    policy_name = "ci-exasol-sagemaker-extension-policy"
+    try:
+        policy_document = cleandoc("""
+        {
+            "Version": "2012-10-17",
+            "Statement": [
+                {
+                    "Effect": "Allow",
+                    "Action": [
+                        "s3:*"
+                    ],
+                    "Resource": "*"
+                }
+            ]
+        }
+        """)
+        response = iam_client.create_policy(
+            PolicyName=policy_name,
+            PolicyDocument=policy_document,
+            Description='This policy is used for the CI Tests of the exasol.sagemaker-extension',
+
+        )
+        return response["Policy"]["Arn"]
+    except iam_client.exceptions.EntityAlreadyExistsException as ex:
+        print("'EntityAlreadyExistsException' exception is handled")
+        sts_client = boto3.client('sts')
+        account_id = sts_client.get_caller_identity()['Account']
+        policy_arn = f'arn:aws:iam::{account_id}:policy/{policy_name}'
+        return policy_arn
+
+
+@dataclasses.dataclass
+class CITestEnvironment:
+    db_conn: pyexasol.ExaConnection
+    aws_s3_bucket: str
+    aws_sagemaker_role: str
+    connection_object_for_aws_credentials: str
+    aws_region: Optional[str] = None
+
+    def __post_init__(self):
+        self.aws_region = os.environ["AWS_DEFAULT_REGION"]
```

Comment:
Should we check if the region is already set? This will overwrite it and may cause unexpected behaviour.